autoload/gitgutter/highlight.vim

```diff
@@ -133,7 +133,7 @@ endfunction
 
 function! s:get_hl(group, what, mode) abort
   let r = synIDattr(synIDtrans(hlID(a:group)), a:what, a:mode)
-  if r ==# ''
+  if index(['', '-1'], r) >= 0
```

Comment:
I'd rather use `if empty(r) || r == -1` instead - no extra list.

Reply:
Updated. I don't think I even grepped outside of the file for code styles, sorry about that.